### PR TITLE
Capitalize the first letter of ”yes” of alert dialog button

### LINKED ISSUE #12788: The first letter of "yes" on the alert dialog of "Empty cards" feature is not capitalized
###### Reproduction Steps

1. Tap the three dot icon on the deck picker screen
2. Tap "Empty cards"
   (Then, a progress dialog of "Finding empty cards..." gets shown)
image
4. Tap Back button during the message is shown
   (Then, an alert dialog of "Do you want to cancel?")

###### Expected Result
Capitalization styles of the buttons of the dialog are consistent (e.g. "[No]   [Yes]", "[NO]   
[YES]"). 
Since all-caps style of buttons has been removed by #12409, "[No]  [Yes]" is expected.

###### Actual Result
Capitalization styles of the buttons of the dialog are inconsistent ("[No]   [**y**es]")
  image

###### Debug info
```
AnkiDroid Version = 2.16alpha89

Android Version = 11

Manufacturer = OPPO

Model = CPH2013

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 11; CPH2013 Build/RKQ1.200903.002; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/106.0.5249.126 Mobile Safari/537.36

ACRA UUID = 7fa108ed-8b79-4927-9c5a-09dd92f52b04

New schema = true

Scheduler = std3

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```
###### Research

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

### PR DESCRIPTION
**Pull request by snowtimeglass**
## Purpose / Description
Capitalize the first letter of ”yes” of alert dialog button of "Empty cards" feature
image

## Fixes
Fixes #12788

## Approach
- Create a new string of "Yes" for alert dialog on 03-dialog.xml
- Replace the current string ("yes") with the new one above ("Yes") on Deckpicker.kt
- Remove the current string ("yes") on 01-core.xml

## How Has This Been Tested?
Checked on a physical device
image

## Checklist

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/res/values/03-dialogs.xml [hunk 1]
```diff
@@ -109,6 +109,7 @@
     <string name="dialog_ok">OK</string>
     <string name="dialog_confirm">Confirm</string>
     <string name="dialog_no">No</string>
+    <stringname="dialog_yes">Yes</string>
```
Comment (snowtimeglass) [inline AnkiDroid/src/main/res/values/03-dialogs.xml]: Typo (already fixed): `stringname`

**Review by lukstbit — APPROVED** [review]
Thanks!

**Comment by snowtimeglass** [comment]
#12791 has been reflected.

**Review by mikehardy — APPROVED** [review]
LGTM and I just cleared strings out via crowdin sync in prep to merge this

**Comment by snowtimeglass** [comment]
Thank you both!

**Pull request merged**